admin/class-ninja-forms-spn-addon-admin.php

```diff
@@ -96,8 +96,8 @@ public function enqueue_scripts() {
 		 * class.
 		 */
 
-		wp_enqueue_script( 'intlTelInput', plugin_dir_url( dirname( __FILE__ ) ) . 'vendor/intl-tel-input-master/build/js/intlTelInput-jquery.js', array(), '17.0.19', true );
-		wp_enqueue_script( 'intlTelInputUtils', plugin_dir_url( dirname( __FILE__ ) ) . 'vendor/intl-tel-input-master/build/js/utils.js', array( 'intlTelInput' ), '17.0.19', true );
+		wp_enqueue_script( 'intlTelInput', plugin_dir_url( dirname( __FILE__ ) ) . 'node_modules/intl-tel-input/build/js/intlTelInput-jquery.js', array(), '18.1.8', true );
+		wp_enqueue_script( 'intlTelInputUtils', plugin_dir_url( dirname( __FILE__ ) ) . 'node_modules/intl-tel-input/build/js/utils.js', array( 'intlTelInput' ), '18.1.8', true );
```

Comment:
We don't need to enqueue this script via `wp_enqueue_script`.
This module should be imported via `@import` JS feature